bower.json

```diff
@@ -2,9 +2,9 @@
   "lib": {
     "name": "bower-rails generated lib assets",
     "dependencies": {
-      "angular": "v1.4.7",
-      "angular-route": "v1.4.x",
-      "angular-resource": "v1.4.8",
+      "angular": "=1.4.9",
```

Comment:
Could we make them consistent? I'm fine with either the = or the v. I think it would be easier for developers to get acquainted with the project